oak_remote_attestation/proto/v1/messages.proto

```diff
@@ -49,10 +51,14 @@ message AttestationEvidence {
 // in response to its request for the enclave's public key(s).
 message AttestationBundle {
   // Attestation evidence from the enclave.
-  AttestationEvidence attestation_evidence = 1;
+  // TODO(#4488): Remove once this is no longer used.
+  AttestationEvidence attestation_evidence = 1 [deprecated = true];
 
   // Supporting evidence required for verifying the integrity of attestation evidence.
   AttestationEndorsement attestation_endorsement = 2;
+
+  // The DICE attestation evidence.
+  oak.attestation.v1.Evidence dice_evidence = 3;
```

Comment:
Also, I think we could call the variable `evidence`, since the corresponding message is just called `Evidence`

Reply:
This is for consistency with `SendAttestationEvidenceRequest`, which already has an existing field called `evidence`.

Reply:
But once we move it out of the bundle and add directly to the response message, we'll call it an `evidence`?

Reply:
Yes